src/native/minipal/guid.h

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef MINIPAL_GUID_H
+#define MINIPAL_GUID_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+    extern "C"
+    {
+#endif // __cplusplus
+
+typedef struct minipal_guid__
```

Comment:
nit:
```suggestion
typedef struct
```

Reply:
We get a slightly better experience in debuggers by giving a name to the struct instead of just naming the typedef (without the name on the struct, you just get "anonymous struct" which is a little annoying).

Reply:
We get a slightly better experience in debuggers by giving a name to the struct instead of just naming the typedef (without the name on the struct, you just get "anonymous struct" which is a little annoying).